.github/.wordlist.txt

```diff
@@ -1053,6 +1056,7 @@ PSCAN
 PSECT
 PSK
 PSoC
+PSRAM
```

Comment:
Maybe we could backtick some of these in readmes. Not sure PSRAM/DTS/BLDevCube should be considered words.

Reply:
DTS/BLDevCube/PSRAM removed from .wordlist.txt and using backticks to escape these words.